src/MetaDataConverterMain.cpp

```diff
@@ -37,7 +37,12 @@ int main (int argc, char** argv) {
                    "this index. Skipping\n";
       continue;
     }
-    convertHmapBasedPermutatationToMmap(permutName, permutName + ".converted",
+    // TODO<joka921> determine magic Number and check if this is necessary at
+    // all
+    /*convertHmapBasedPermutatationToMmap(permutName, permutName + ".converted",
                                         permutName + MMAP_FILE_SUFFIX);
+                                        */
+    addBlockListToMmapMetaDataPermutation(permutName,
```

Comment:
Here I will add the automatic version detection so that everything is correctly converted and only if  necessary